### PR TITLE
[Improvement] Adds documentation for new API filters for operations and clearings

### PR DESCRIPTION
**Pull request by santi698**
Documentation for https://github.com/IncreaseCard/increasecard/pull/918

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1557,6 +1561,14 @@ Retrieves all the different Clearings card types for the current client.
         Filter by date range.
     + shop_id (optional, string)
         Filter by shop id
+    + stablishment_id (optional, string)
+        Filter by stablishment id
+    + priority (optional, integer)
+        Filter by priority
+    + created_at_from (optional, date)
```
Comment (aguspagnoni) [inline apiary.apib]: I would add date valid format

Comment (santi698) [inline apiary.apib]: There's the standard ISO 8601 format used in most APIs and JSON do you think it is still not clear? Now that I think about it we are not using the standard in other parameters of the API, we should

Comment (santi698) [inline apiary.apib]: We should stick to the standard but Rails accepts most date formats anyway

File: apiary.apib [hunk 2]
```diff
@@ -1557,6 +1561,14 @@ Retrieves all the different Clearings card types for the current client.
         Filter by date range.
     + shop_id (optional, string)
         Filter by shop id
+    + stablishment_id (optional, string)
+        Filter by stablishment id
+    + priority (optional, integer)
+        Filter by priority
+    + created_at_from (optional, date, `2017-01-01T12:00:00+01:00`)
```
Comment (santi698) [inline apiary.apib]: @aguspagnoni I added an example created_at_from for clarity, what do you think?

Comment (aguspagnoni) [inline apiary.apib]: I would also add, if posible, that we recommend using `iso8601` format. Users will thank us for being transparent on that

**Pull request merged**